main.tf

```diff
@@ -0,0 +1,166 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5.16"
+    }
+  }
+
+  required_version = ">= 1.5.7"
+
+  backend "s3" {
+    bucket               = "ccao-terraform-state-us-east-1"
+    key                  = "workspaces/default/terraform.tfstate"
+    region               = "us-east-1"
+    workspace_key_prefix = "workspaces"
```

Comment:
This block defines the [backend](https://developer.hashicorp.com/terraform/language/settings/backends/configuration) that Terraform will use to manage state. The `key` attribute defines the location of the default workspace, but we don't use that workspace at all in our GitHub Actions config, so workspace state files will instead follow the naming scheme set by `workspace_key_prefix` and will be stored under `s3://ccao-terraform-state-us-east-1/workspaces/<workspace>/terraform.tfstate`.